projects/storefrontlib/src/cms-pages/product-detail-page/product-detail-page.component.ts

```diff
@@ -0,0 +1,7 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'cx-product-dertail-page',
```

Comment:
typo in the word `detail`. 

Btw. I've seen in other places we use plural `product-details`. For consistency, maybe it would be good to stick with it?